src/main/java/com/footbalmanager/app/services/LeagueServiceImpl.java

```diff
@@ -33,4 +44,9 @@ public void delete(Long id) {
     public void delete(League entity) {
         leagueRepository.delete(entity);
     }
+
+    @Override
+    public void save(PostLeagueRequestDto dto) {
```

Comment:
O co chodzi z tymi DTO do czego nam są potrzebne?

Reply:
To co otrzymujesz jako json z requestu, wpisz data transfer object w google ;d

Reply:
:D wiem co to DTO, tylko zastanawiałem się gdzie je potrzebujemy. Dzięki za info :)

Reply:
Mógłbym w kontrolerze zmieniać DTO na np. League, ale wydaje mi się, że bardziej elegancko poprostu wrzucić dto do serwisu i niech się on martwi stworzeniem encji. Używam zawsze notacji dto bo przynajmniej wiadomo skąd się biorą obiektu (...RequestDTO -> jsony przychodzące do API, ...ResponseDTO -> jsony wychodzące z api)

Reply:
DTO spoko tylko to powielanie kodu na request i response średnio mi się podoba bo mamy dwa obiekty do utrzymania które mają te same pola. Wbrew regule "Don't repeat yourself" DRY :D

Reply:
No ale nie zawsze masz te same pola w requescie i response? Zostaly stworzone po to zeby je z biegiem pisania kodu uzupełnic. Moge stosowav hashmape ale wzorce po cos jednak zostaly stworzone chyba :) nawet kosztem nadmiarowego kodu

Reply:
OKej :D zostawiamy.